co2eq/purchase/index.js

```diff
@@ -10,10 +10,18 @@ import {
   UNIT_ITEM,
   UNIT_LITER,
 } from '../../definitions';
-import { convertToEuro, getAvailableCurrencies } from '../../integrations/utils/currency/currency';
 import { getChecksum } from '../utils';
 import footprints from './footprints.yml';
 import consumerPriceIndex from './consumerpriceindices.yml';
+import exchangeRates2011 from './exchange_rates_2011.json';
+
+export function convertTo2011Euro(amount, currency) {
+  if (!Object.keys(exchangeRates2011.rates).includes(currency.toUpperCase())) {
```

Comment:
```suggestion
  if (exchangeRates2011.rates[currency.toUpperCase()] != null) {
```
as this will not only check that the key is here, but also that the value is not null